aiida/engine/processes/control.py

```diff
@@ -93,7 +93,7 @@ def play_processes(
     :raises ``ProcessTimeoutException``: If the processes do not respond within the timeout.
     """
     if not get_daemon_client().is_daemon_running:
-        raise DaemonException('The daemon is not running.')
+        LOGGER.warning('The daemon is not running, so some processes may not be reachable.')
```

Comment:
Maybe we can be explicit then?

```suggestion
        LOGGER.warning('The daemon is not running, so processes submitted to the daemon workers are not reachable.')
```

Reply:
I guess, but it is still not technically correct. If you run `verdi daemon worker`, it will start an actual daemon worker, just not one that is managed by the circus daemon. And if you then submit a process, it will get picked up by that worker and so it will respond to pause/play/kill, even though the "actual daemon" is not running.

But I guess this is an edge-case that is not really going to be happening often, so maybe this somewhat incorrect clarification is still going to be useful in most cases.

Reply:
I added the suggestion but removing "workers" i.e.
```
The daemon is not running, so processes submitted to the daemon are not reachable.
```